src/style.css

```diff
@@ -0,0 +1,132 @@
+* {
+  box-sizing: border-box;
+}
+
+.home-page {
+  font-family: Roboto, sans-serif;
+  --color-text-1: #060b35;
+  --color-text-2: #616070;
+}
+
+.home-page__content {
+  margin: 0;
+}
+
+.card {
+  width: 200px;
+  display: flex;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__image {
+  height: 134px;
+  margin: 32px 19px 40px;
+}
+
+.card__name {
+  line-height: 18px;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  color: var(--color-text-1);
+  margin: 0 16px 4px;
```

Comment:
apply paddings to the whole card instead of adding horizontal margins for each inner element
<img width="254" alt="image" src="https://user-images.githubusercontent.com/66749919/232112339-5f26bd43-2412-4b80-9e2e-24339d7fd622.png">